docs/octopus-cloud/Uptime SLO

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
this page will need to end in .md e.g. `uptime-slo.md`

Reply:
Done, thank you for your patience. I am sure you have guessed this is my first-time using GitHub as I am not a developer.

Reply:
No worries, I updated the page again to lower case and removed spaces :)